pkg/debug/scripts/collect-info.sh

```diff
@@ -369,6 +369,23 @@ collect_kube_info()
         } > "$DIR/kube-info"
     fi
 }
+
+collect_longhorn_info()
+{
+    type=$(cat /run/eve-hv-type)
+    if [ "$type" != "kubevirt" ]; then
+        return
+    fi
+    echo "- Collecting Longhorn specific info"
+    {
+        echo "  - longhorn support bundle "
+        # Longhorn issue reports have seen hangs generating this data
+        # when nodes are down, or due to longhorn bugs.
+        # Give up after 5min, and allow remaining system data to be collected.
```

Comment:
How large can this be?
5 min. seems like a lot of time compared to a typical duration of collect-info.sh. If stuck, user would likely loose patience and kill the whole script. Can we lower the timeout to seconds?

Reply:
Some of the variability can be from the pull of the support bundle image.  On a three node system I just saw 132 seconds for an initial run due to also pulling the support bundle image, and a later run was only 45 seconds.   This is a good point i'll cleanup the comments a bit and change the echo to inform the user to be patient as this step can take a longer time.

Reply:
@milan-zededa changes added to help address users wait time expectation.  I'm afraid the timeout would have to be dropped to something closer to 30-60 seconds to fit in with the existing time spans.  To get closer to this (with room to spare) we will need to start by including the support image in the system, I will investigate this option for a later PR.